lib/autokey/gtkapp.py

```diff
@@ -171,12 +171,20 @@ def hotkey_removed(self, item):
     def path_created_or_modified(self, path):
         time.sleep(0.5)
         changed = self.configManager.path_created_or_modified(path)
+        if not self.monitor.has_watch(path) and os.path.isdir(path): 
```

Comment:
These sections could probably be extracted as `add_open_file_watch` and `remove_open_file_watch` functions